modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/item_selector_preview/js/ItemSelectorPreview.es.js

```diff
@@ -101,7 +101,7 @@ const ItemSelectorPreview = ({
 	};
 
 	const handleClickEdit = () => {
-		const itemTitle = currentItem.title;
+		const itemTitle = JSON.parse(currentItem.value).title;
```

Comment:
Hey @NemethNorbert, I found this very tricky. In the future we will forget why are we doing this complex calculation instead of just reading the "title" field of the currentItem, which is logical. 
I prefer what you did in the first commit, ItemSelector should receive the original title and not force to add the extension. This way is more consistent with what we have in Document & Media, as this tab in the item selector should be a mirror of it.

Reply:
Completely agree, thank you :)